### ISSUE TITLE
The Rope doesnt tear when the terrain explodes underneath

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/hedgewars#40]
```
What steps will reproduce the problem?
1. Shoot a rope to a mine
2. Arm the mine
3. Move back with the rope

What is the expected output? What do you see instead?
The Rope to tear

What version of the product are you using? On what operating system?
0.9.13, Win7

Please provide any additional information below.
http://www.youtube.com/watch?v=iPOKU5iA-Pw

```

-----
Original issue reported on code.google.com by `joship...@gmail.com` on 15 Sep 2010 at 10:26

**Comment by GoogleCodeExporter** [comment]
```
Issue 38 has been merged into this issue.
```

-----
Original comment by `mario.li...@gmail.com` on 16 Sep 2010 at 9:08

**Comment by GoogleCodeExporter** [comment]
```
One of the issues we didn't touch so far as at least some consider it being a 
part of the game (i.e. intended). Personally I'm not sure about it and tearing 
the rope might be an option but that's nothing I'd decide right away without 
more opinions.

Setting the status to "Accepted" for now but this doesn't mean we'll take 
action immediately (if at all).
```

-----
Original comment by `mario.li...@gmail.com` on 16 Sep 2010 at 9:11
* Changed state: **Accepted**
* Added labels: **Priority-Low**
* Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
Similar to the crate+rope bug he filed.  Same basic behaviour. One of these 
should be duped against the other, although some of the comments in the crate 
bug suggested it had gotten mixed up w/ tangling on small bits of land.

Anyway, the thread in forum on this is probably a good place to continue the 
discussion.

```

-----
Original comment by `kyberneticist@gmail.com` on 21 Sep 2010 at 10:00

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 4386:ed78465973f6

```

-----
Original comment by `kyberneticist@gmail.com` on 21 Nov 2010 at 6:21
* Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Verified (mine breaks rope)
```

-----
Original comment by `jerryma1...@gmail.com` on 26 Nov 2011 at 7:58
* Changed state: **Verified**
